hasher-matcher-actioner/hmalib/lambdas/api/submit.py

```diff
@@ -99,12 +109,38 @@ def submit(
             return SubmitContentResponse(
                 content_id=request.content_id, submit_successful=True
             )
-
-        # Other possible submission types are not supported so just echo content_id for testing
-        bottle.response.status = 422
-        return SubmitContentError(
-            content_id=request.content_id,
-            message="submission_type not yet supported",
-        )
+        elif request.submission_type == "URL":
+            fileName = request.content_id
+            url = request.content_ref
+            response = requests.get(url)
+            # TODO better checks that the URL actually worked...
+            if response and response.content:
+                # TODO a whole bunch more validation and error checking...
+
+                # Right now this makes a local copy in s3 but future changes to
```

Comment:
This is great. You have identified the path of least resistance. I would say how the submit api and the hasher lambdas works together is an internal impl detail. It is perfectly okay to NOT give the api/user an option.